locations/spiders/deichmann.py

```diff
@@ -13,29 +14,23 @@ class DeichmannSpider(SitemapSpider, StructuredDataSpider):
         "https://stores.dosenbach.ch/sitemap.xml",
     ]
     sitemap_rules = [
-        (
-            r"https://stores\.dosenbach\.ch/[-\w]+/[-\w]+/[-\w]+\.html",
-            "parse_sd",
-        ),  # CH
-        (
-            r"https://stores\.deichmann\.com/[-\w]+/[-\w]+/[-\w]+\.html",
-            "parse_sd",
-        ),  # DE
-        (
-            r"https://stores\.deichmann\.com/\w\w-\w\w/[-\w]+/[-\w]+/[-\w]+\.html",
-            "parse_sd",
-        ),  # All the others
+        (r"^https:\/\/stores\.deichmann\.com\/[a-z]{2}-[a-z]{2}\/[a-z]{2}(?:\/[-\w]+){3}$", "parse_sd"),
+        (r"^https:\/\/stores\.dosenbach\.ch\/ch-de\/ch(?:\/[-\w]+){3}$", "parse_sd"),
     ]
-    wanted_types = ["ShoeStore"]
-
-    def sitemap_filter(self, entries):
-        for entry in entries:
-            # Filter out excessive matches
-            if not entry["loc"].endswith("index.html"):
-                yield entry
+    wanted_types = ["LocalBusiness"]
 
     def post_process_item(self, item, response, ld_data, **kwargs):
         if "dosenbach.ch" in response.url:
             item.update(self.DOSENBACH)
 
+        item["ref"] = item["ref"].split("#")[1]
+
+        item["opening_hours"] = OpeningHours()
+        item["opening_hours"].from_linked_data(ld_data)
+
```

Comment:
Since you've fixed `from_linked_data` is this still needed?

```suggestion
```

Reply:
Thanks I fixed it. I forgot to remove that after fixing from_linked_data to work with international words for "CLOSED"